share/lang/fr.json

```diff
@@ -160,59 +160,60 @@
   "config": {
     "style": {
       "_": "Style",
-      "resize-factor": "Echelle d'interface",
+      "resize-factor": "Échelle d'interface",
       "body-margin": "Marge extérieur",
       "body-font": "Police",
       "nav-opacity": "Opacité de barre de navigation lorsqu'elle est inactive",
       "accent-color": "Accentuation",
       "shadow-card": "Fenêtre",
       "shadow-text": "Texte",
       "content-align": "Alignement",
-      "font-size-small": "'Petit' Taille de la police",
+      "font-size-small": "'Petit' taille de la police",
       "gauge-height": "Taille",
       "graph-height": "Taille des lignes du tableau"
     },
     "color": {
-      "_": "Guage Color",
-      "gauge-default": "Defaut",
+      "_": "Jauge des couleurs",
+      "gauge-default": "Défaut",
       "gauge-opacity": "Opacité",
       "position-tank": "Tank",
       "position-deal": "DPS",
       "position-heal": "Soin",
       "pld": "Paladin (Gladiateur)",
       "war": "Guerrier (Maraudeur)",
-      "drk": "Chevalier Noir",
+      "drk": "Chevalier noir",
       "gnb": "Pistosabreur",
       "mnk": "Moine (Pugiliste)",
-      "drg": "Dragonier (Lancier)",
+      "drg": "Chevalier dragon (Lancier)",
       "brd": "Barde (Archer)",
-      "nin": "Ninja (Voleur)",
+      "nin": "Ninja (Ninja)",
```

Comment:
`"nin": "Ninja (Surineur)",` ?

Reply:
You'r right, i have updating my PR. thank you